NEWS.md

```diff
@@ -1,3 +1,7 @@
+## 20.2.0 2021-04-23
+
+* Determines shelving order for existing items during tenant upgrade (MODINVSTOR-521)
```

Comment:
As this is a minor release made from a long lived release branch, please make sure the mainline is updated to use `20.3.0` as the next release version to avoid a future conflict in versions

Reply:
Thanks, just created https://github.com/folio-org/mod-inventory-storage/pull/614